cmd/cli/uninstall_mesh_test.go

```diff
@@ -542,6 +567,10 @@ func TestUninstallClusterWideResources(t *testing.T) {
 				Log:          func(format string, v ...interface{}) {},
 			}
 
+			fakeClientSet := fake.NewSimpleClientset(existingKubeClientsetObjects...)
+			_, err = addDeployment(fakeClientSet, "osm-controller-1", testMeshName, testNamespace, osmTestVersion, true)
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
Let's stick with any new checks in these tests using `testify` instead. Then the `RegistryFailHandler` line above can be removed too.